docs/examples/workflow/README.md

```diff
@@ -1,130 +1,111 @@
-    default       foowf-v1         1          83cf4e8246a89afa   WorkflowStep
-    ```
 
 1. Apply Application:
 
     ```
     kubectl apply -f app.yaml
     ```
 
-1. Check workflow status in Application:
-
-    ```
-    kubectl get application first-vela-app -o=jsonpath='{.status.workflow[?(@.name=="my-wf")]}.phase'
-    ```
 
-    Output:
-    ```
-    running
-    ```
 
-1. Check Workflow objects:
+2. Check workflow status in Application:
 
     ```
-    kubectl get foo my-wf -o=jsonpath='{.spec.key}'
+    kubectl get -f app.yaml
     ```
 
     Output:
-
-    ```
-    test
-    ```
-
-    This means the resource has been rendered correctly.
-
-1. Check workflow context:
-
-    ```
-    kubectl get foo my-wf -o=jsonpath='{.metadata.annotations.app\.oam\.dev/workflow-context}' | jq
+    ```yaml
+    ...
+    status:  
+      workflow:
+        appRevision: application-sample-v1
+        contextBackend:
+          apiVersion: v1
+          kind: ConfigMap
+          name: workflow-application-sample-v1
+          uid: 783769c9-0fe1-4686-8528-94ce2887a5f8
+        stepIndex: 2
+        steps:
+        - name: deploy-server1
+          phase: succeeded
+          resourceRef:
+            apiVersion: ""
+            kind: ""
+            name: ""
+          type: apply
+        - name: deploy-server2
+          phase: succeeded
+          resourceRef:
+            apiVersion: ""
+            kind: ""
+            name: ""
+          type: apply
+
+      ```
+
+2. Check Resource in cluster.
+
+    ```
+    kubectl get pods
     ```
 
     Output:
 
-    ```json
-    {
-      "appName": "first-vela-app",
-      "appRevision": "first-vela-app-v1",
-      "workflowIndex": 0,
-      "resourceConfigMap": {
-        "name": "first-vela-app-v1"
-      }
-    }
-    ```
-
-1. Patch condition status on workflow object:
-
-    ```
-    kubectl patch foo my-wf --type merge --patch "$(cat wf-patch.yaml)"
-    ```
-
-    Check workflow object status:
-
     ```
-    kubectl get foo my-wf -o=jsonpath='{.status.conditions[?(@.type=="workflow-finish")]}' | jq
+    NAME       READY   STATUS    RESTARTS   AGE
+    firstApp   1/1     Running   0          15s
+    secondApp  1/1     Running   0          18s
     ```
 
-    Output:
-
-    ```json
-    {
-      "message": "{\"observedGeneration\":2}",
-      "reason": "Succeeded",
-      "status": "True",
-      "type": "workflow-finish"
-    }
-    ```
+    This means the resource has been rendered correctly.
 
-    > Note: The observedGeneration is 2 because the json patch will trigger generation increment.
 
-1.  Check workflow status in Application:
+3. Check `secondApp` Environment variable
 
     ```
-    kubectl get application first-vela-app -o=jsonpath='{.status.workflow[?(@.name=="my-wf")]}.phase'
+    kubectl exec secondApp -- env|grep PrefixIP
     ```
 
     Output:
+
     ```
-    succeeded
+    PrefixIP=10.244.0.22
     ```
+## WorkflowStep Definition Introduction.
```

Comment:
We should put this into a single doc and show it on website too.
Maybe use a table to describe the field?